librz/core/cmd_descs.h

```diff
@@ -51,6 +51,47 @@ RZ_IPI RzCmdStatus rz_cmd_help_search_handler(RzCore *core, int argc, const char
 RZ_IPI int rz_cmd_help(void *data, const char *input);
 RZ_IPI RzCmdStatus rz_push_escaped_handler(RzCore *core, int argc, const char **argv);
 RZ_IPI int rz_cmd_ox(void *data, const char *input);
+RZ_IPI RzCmdStatus rz_analysis_function_blocks_list_handler(RzCore *core, int argc, const char **argv, RzOutputMode mode);
```

Comment:
Same here - maybe it's better to separate the `RZ_IPI RzCmdStatus` functions from the rest?

Reply:
this is auto-generated, no need for a separator or anything.